### ISSUE TITLE
Feature request - buttons size customization

### ISSUE THREAD

**Issue by eugeniy-balaban** [osmandapp/OsmAnd#6997]
It would be convenient to be able to customize the size of the buttons on the main screen. May be separately for each profile. For example when using gloves it is more convenient to have large buttons.

**Comment by scottbmc** [comment]
Yes, please add this feature. Also nice would be a left handed button layout or a fully configurable button layout.

**Comment by AnastasiaTiutina** [comment]
![IMG_E6BEDBB420A3-1 (1)](https://user-images.githubusercontent.com/72389175/98230764-33bac180-1f64-11eb-8de1-bd7f27d0e99f.jpeg)
Make zoom buttons bigger like in attachment for motorbike profile. It is very difficult while driving the motorbike and wearing winter gloves to hit this button properly in the actual size (which is a perfect size for normal use but way too small when using gloves)

**Comment by flobee** [comment]
YES! i need but buttons!

is there no config to handle things like that? 
depending on the profile the look and feel should be customizable. 
Compare "Google Auto" and Routing function of "Google Maps". Google Auto more save against accidents while driving.

E.g general button size 1-5  where 3 is standard. 
Some navi manufactures have those settings.

**Comment by mid-kid** [comment]
I want to make the buttons and text on the main view *smaller* so I can see more of the map. They take too big a portion of the screen.

**Comment by flobee** [comment]
> I want to make the buttons and text on the main view _smaller_ so I can see more of the map. They take too big a portion of the screen.

hopefully you dont get old and need glasses :-) like me

**Comment by mid-kid** [comment]
Haha, I sure hope not, but getting a phone with a slightly different resolution caused the buttons to balloon in size for some reason.

**Comment by mikehgentry** [comment]
I *think* it's affected by the UI size in Android settings, so you can try adjusting that if the side effects in other programs aren't too severe.

**Comment by dmpr0** [comment]
Done https://github.com/osmandapp/OsmAnd/issues/21815
